common/api/map-layers-formats.api.md

```diff
@@ -56,6 +58,17 @@ export class ArcGisFeatureProvider extends ArcGISImageryProvider {
     get tileSize(): number;
 }
 
+// @beta
+export interface CreateSessionOptions {
```

Comment:
This interface and the LayerTypes and MapTypes types it references seem to be specific to GoogleMaps yet they all have very generic names, are they supposed to be applicable to other map imagery providers?

Reply:
It's meant to be GoogleMaps specific.  Initially it contained the 'GoogleMaps' prefix but found that long and redundant.  So better to have the GoogleMapsXYZ terminology ?

Reply:
Yes, the names are going into a global scope so they need to indicate they are specific to google maps.

Reply:
Using a namespace is not an option right?

Reply:
It doesn't solve your problem of names being too long. If you put them in a `GoogleMaps` namespace then the name becomes `GoogleMaps.LayerType` instead of `GoogleMapsLayerType`, not an improvement IMO and namespaces have other downsides.

Reply:
done, added'GoogleMaps' prefix